pages/apim/3.x/installation-guide/installation-guide-plugins.adoc

```diff
@@ -0,0 +1,187 @@
+:page-folder: apim/overview
+:page-layout: apim3x
+:page-liquid:
+:page-description: Gravitee.io API Management - Installation Guide - Plugins
+:page-keywords: Gravitee.io, API Management, apim, guide, installation, upgrade, plugins
+:page-toc: true
+
+// author: Tom Geudens
+== Overview
+Plugins bring Gravitee APIM to life. They provide functionality ranging from connectors (eg. Kafka) to tracers (eg. Jaeger) and everything in between. A special mention goes to policies, plugins that provide you with options to shape and control the flow of your APIs.
+
+A lot of plugins are part of the Gravitee APIM installation. Ones that aren't generally applicable can be added. You can even write your own! This document explains how you can install and upgrade additional plugins.
+
+== In a nutshell - Option I
+. Get your plugin zip-file ready.
++
+The supported plugins for Gravitee APIM can be found link:https://download.gravitee.io/#graviteeio-apim/plugins/[here, window=\"_blank\"].
+
+. Drop the plugin zip-file in the plugins-folder of the relevant component. This could be the APIM REST API, the APIM Gateway or both!
+
+. Restart the component.
+
+NOTE: At this time installing and updating plugins requires a restart of the component.
+
+== In a nutshell - Option II
+. Get your plugin zip-file ready.
++
+The supported plugins for Gravitee APIM can be found link:https://download.gravitee.io/#graviteeio-apim/plugins/[here, window=\"_blank\"].
+
+. Drop the plugin zip-file in a plugins-folder of your own choosing.
+
+. Modify the `gravitee.yml` configuration file of the relevant component (APIM REST API / APIM Gateway / both) to take the additional folder into account
+
+. Restart the component.
+
+NOTE: At this time installing and updating plugins requires a restart of the component.
+
+== Details for .ZIP installations
+=== Option I
+. Locate the plugins folder of the component
++
+*  APIM REST API - <your installation folder>/graviteeio-apim-rest-api-{{ site.products.apim._3x.version }}/plugins
+*  APIM Gateway - <your installation folder>/graviteeio-apim-gateway-{{ site.products.apim._3x.version }}/plugins
+
+. Drop the plugin zip file into the folder
+
+. Restart the component
+
+=== Option II
+. Create a custom location for plugins
++
+[source,bash]
+----
+mkdir /customlocation/plugins
+----
++
+NOTE: This is an example location, yours can differ.
+
+. Drop the plugin zip-file into that folder
+
+. Modify the `gravitee.yml` configuration file of the component
++
+*  APIM REST API - <your installation folder>/graviteeio-apim-rest-api-{{ site.products.apim._3x.version }}/config
+*  APIM Gateway - <your installation folder>/graviteeio-apim-gateway-{{ site.products.apim._3x.version }}/config
+
++
+[source,yaml]
+----
+# Plugins repository
+plugins:
+  path:
+    - ${gravitee.home}/plugins
+    - /customlocation/plugins
+----
+
+. Restart the component
+
+== Details for package installations (Amazon Linux, RHEL, CentOS)
```

Comment:
I don't see any real difference in instructions for :
- in a nutshell
- zip installations
- package installations

Maybe we could some common instructions ?
(Just a suggestion, maybe it's more clear for user to distinct those cases)

Reply:
That's a good comment. The "in a nutshell" quickly describes the two basic options (put the zip file in the installation folder or put the zip file in a customer folder and modify gravitee.yml). 

Then I went through the deployment options we have in the rest of our documentation (zip, package, docker, kubernetes) describing how they are applied in each.

That is quite a bit of repetition and the differences between zip and package are minimal, agreed.